stdlib/public/core/StringCore.swift

```diff
@@ -660,8 +660,8 @@ extension _StringCore : RangeReplaceableCollection {
       let tailStart = rangeStart + (replacedCount &<< elementShift)
 
       if growth > 0 {
-        (tailStart + (growth &<< elementShift)).copyBytes(
-          from: tailStart, count: tailCount &<< elementShift)
+        (tailStart + (growth &<< elementShift)).copy(
+        from: tailStart, bytes: tailCount &<< elementShift)
```

Comment:
Here and below, please restore the original spacing, which indents two spaces when breaking after an opening parenthesis.